server/svix-server/src/core/otel_spans.rs

```diff
@@ -60,8 +60,8 @@ impl<B> MakeSpan<B> for AxumOtelSpanCreator {
 
         let request_id = request
             .extensions()
-            .get::<RequestId>()
-            .and_then(|id| id.header_value().to_str().ok())
+            .get::<KsuidMs>()
+            .map(ToString::to_string)
```

Comment:
Why do we need this extra struct and request_ids.rs file? Why not literally just do:
```rust
let request_id = KsuidMs::generate(None ,None);
```
?